### PR TITLE
Feature/irrigation dates filter

### PR DESCRIPTION
**Pull request by Adamws33**
updated the sidebar to have a filter to display the Colorado irrigation dates.

### CODE REVIEW

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://4.255.50.46:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://4.255.50.46:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

File: src/pages/CropSidebar/CropSidebar.js [hunk 1]
```diff
@@ -95,6 +97,13 @@ const CropSidebar = ({
     setShowFilters(value);
   }, [speciesSelectorActivationFlagRedux, from, comparisonView]);
 
+  const handleIrrigationFilter = () => {
+    dispatchRedux(setIrrigationFilter(!irrigationFilterRedux));
+  };
+
+  useEffect(() => {
+  }, [irrigationFilterRedux]);
```
Comment (YazidZidane) [inline src/pages/CropSidebar/CropSidebar.js:105]: This is an empty useEffect

**Comment by YazidZidane** [comment]
I don't have much background information on this, but I think if the filter text style is aligned to the followings it might be better.
image

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://4.255.50.46:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://4.255.50.46:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://4.255.50.46:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://4.255.50.46:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://4.255.50.46:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Review by YazidZidane — APPROVED** [review]

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.242.236.25:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.242.236.25:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Pull request merged**